src/components/PageHeader.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './PageHeader.css'
+import PropType from 'prop-types'
+import { isValidUrl } from '../utils/index'
+
+const PageHeader = props => {
```

Comment:
typically you'll want to unwrap the object: `const PageHeaer = ({ title, background, etc }) =>`